ui/js/perfapp.js

```diff
@@ -8,7 +8,7 @@ perf.config(function($compileProvider, $stateProvider, $urlRouterProvider) {
     $stateProvider.state('graphs', {
         title: 'Perfherder Graphs',
         templateUrl: 'partials/perf/graphsctrl.html',
-        url: '/graphs?timerange&series&highlightedRevisions&zoom',
+        url: '/graphs?timerange&series&highlightedRevisions&zoom&tooltip',
```

Comment:
Can you rename `tooltip` here to `selected`? Tooltip is too vague, there are many things in the graphs view that are tooltips. :)